docs/dataservice-write-example.md

```diff
@@ -77,6 +77,28 @@ After installing and running the `dataservice_write_example` apk, you should see
 
 Note, that you can run `CMake` command directly from `<olp-edge-sdk-root>/examples/dataservice-write/` folder if you have already built and installed OLP C++ EDGE SDK libraries for Android. Make sure that you pass the correct path to `LevelDB` library and provided the correct `EDGE_SDK_NETWORK_PROTOCOL_JAR` parameter in the `CMake` command invoked by the `build/examples/dataservice-write/android/app/build.gradle` script.
 
+## Building and running on iOS
+
+This example shows how to integrate and use the OLP C++ EDGE SDK in a basic iOS application written in Objective-C language.
+
+Configure the OLP C++ EDGE SDK with `EDGE_SDK_BUILD_EXAMPLES` variable set to `ON`, disable tests via `EDGE_SDK_ENABLE_TESTING` variable, and specify path to the iOS toolchain file shipped with SDK (located in `<olp-edge-sdk-root>/cmake/toolchains/iOS.cmake`):
+
+```bash
+mkdir build && cd build
+cmake .. -GXcode  -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/iOS.cmake -DPLATFORM=iphoneos -DEDGE_SDK_BUILD_EXAMPLES=ON -DEDGE_SDK_ENABLE_TESTING=OFF
+```
+
+(note that in order to configure the OLP C++ EDGE SDK for a simulator, set `SIMULATOR` variable to `ON`)
+
+Now open the generated `XCode` project:
+```bash
+open olp-cpp-sdk.xcodeproj
+```
+
+Select the `dataservice-write-example` scheme from the schemes list in `XCode` and specify correct your team identifier for the `dataservice-write-example` target.
```

Comment:
```suggestion
Select the `dataservice-write-example` scheme from the schemata list in `XCode` and specify your correct team identifier for the `dataservice-write-example` target.
```

Reply:
I changed a bit this sentence. Btw, what wrong with schemes list? Schemata is plural for schema, but XCode uses **scheme** notion:  https://developer.apple.com/library/archive/featuredarticles/XcodeConcepts/Concept-Schemes.html

Reply:
I thought it was meant to be schema.
Ignore my comment then.